composer.json

```diff
@@ -34,16 +34,12 @@
         "symfony/console": "^4.4.16 || ^5.4 || ^6.0",
         "symfony/stopwatch": "^4.4 || ^5.4 || ^6.0"
     },
-    "conflict": {
-        "doctrine/orm": "<2.12"
-    },
     "require-dev": {
         "ext-pdo_sqlite": "*",
         "doctrine/coding-standard": "^9",
         "doctrine/orm": "^2.12",
         "doctrine/persistence": "^2 || ^3",
         "doctrine/sql-formatter": "^1.0",
-        "ergebnis/composer-normalize": "^2.9",
```

Comment:
I removed that dependency and its configuration because I think no-one is going to break the composer.json so badly that they will need to fix it with that tool as opposed to manually.

Reply:
Sounds reasonable. As I can see we didn't implement it as an dependency in other projects. Its impact wasn't that big for us.